CSharp/VirtoCommerce.Module.Template/Web/Controllers/Api/MyModuleController.cs

```diff
@@ -8,6 +9,7 @@ public class $ext_supersafename$Controller : ApiController
         // GET: api/$ext_supersafename$
         [HttpGet]
         [Route("")]
+        [CheckPermission(Permission = "$ext_safeprojectname$:read")]
```

Comment:
Move all permissions to dedicated class like this:
https://github.com/VirtoCommerce/vc-module-inventory/blob/b38757efde7846315ffa6bd81b626e605bbec82b/VirtoCommerce.InventoryModule.Web/Security/InventoryPredefinedPermissions.cs

Reply:
Permission constants should be declared in the Core project. This will allow to create a NuGet package and reuse these constants in custom modules which override/extend this module.

Reply:
The question is, how do we do it in the end?

Reply:
ok, new sample https://github.com/VirtoCommerce/vc-platform-core/blob/1b0dbdbe4e6eaae1c4c95af559475a4cf93661f5/Modules/vc-module-inventory/VirtoCommerce.InventoryModule.Core/ModuleConstants.cs